### PR TITLE
Adding Food Truck Locator and First Story

### PR DESCRIPTION
**Pull request by code-11**
Let me know if there's anything we should do.

### CODE REVIEW

File: people/brendan-ritter.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Brendan Ritter
+
```
Comment (amfleming) [inline people/brendan-ritter.md:2]: Intentional?

**Comment by amfleming** [comment]
- Looks like you committed something you did not intend in `brendan-ritter.md`
- The rest of this pull request looks good; please rebase / squash the 7 commits into one
- Is this your first story?  `Get something working that is useful`

**Comment by code-11** [comment]
Great. Thanks for the feedback. These changes are adding the link to our project management page. The code for the first story is on my repo (perhaps we misunderstood how you wanted things) There is a link to it on the Pivotal Tracker and also [here](https://github.com/code-11/FoodTruckLocator).

Will squash comments.

**Comment by code-11** [comment]
Ok. I think I squashed correctly. The change to the user page was to fix a merge conflict where I had one less hyphen for the horizontal line in markdown. I guess I should have left it out of this particular pull request.

**Comment by amfleming** [comment]
So when I review the PR there are 9 commits (there should be only 1)

Also the first story in Pivotal (I'm able to see it) is insufficient

**Comment by code-11** [comment]
Sorry, I think there was a misunderstanding about what the first story meant. I think the description of the task should reflect what you expect. 

As for the commits, I admit I'm a bit confused. The first commit (Added myself and...) shouldn't even be there since I sent that through a prior pull request. Secondly, when I squashed the commits into one (Added food truck locator project...) it added it to the list of commits for the pull request instead of replacing them (along with an additional merge commit?) 

Perhaps we can talk before or during class about how to fix this? 
Thanks for the help

**Comment by dblock** [comment]
@dagoch or @karnseth will be happy to help you with all of this! Connect with one of them.

**Comment by dblock** [comment]
This might also be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

**Comment by code-11** [comment]
Going to close this one, reclone the repo and try again.

**Pull request closed without merging**